popups/log_graph_view.html

```diff
@@ -0,0 +1,31 @@
+<html>
+<style>
+{css}
+</style>
+<body>
+<div class="header-wrapper"><h2>Keyboard Shortcuts</h2></div>
+
+<h3>Actions</h3>
+<ul>
+  <li><code><span class="shortcut-key">enter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>view commit</code></li>
```

Comment:
Does `minihtml` support any sort of layout through CSS?  If there isn't any way to align columns without including spaces, this is fine.  But if it can be done with CSS, all the better.